rpc/session.go

```diff
@@ -0,0 +1,21 @@
+package rpc
+
+import (
+	"net"
+	"net/rpc"
+	"net/rpc/jsonrpc"
+)
+
+type Session struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Session should have comment or be unexported